src/common/Database/MySQLConnection.cpp

```diff
@@ -107,36 +112,59 @@ bool MySQLConnection::Open()
     }
     #endif
 
-    m_Mysql = mysql_real_connect(mysqlInit, m_connectionInfo.host.c_str(), m_connectionInfo.user.c_str(),
-        m_connectionInfo.password.c_str(), m_connectionInfo.database.c_str(), port, unix_socket, 0);
-
-    if (m_Mysql)
-    {
-        if (!m_reconnecting)
+    // Possible improvement for future: make ATTEMPTS and SECONDS configurable values
+    uint const ATTEMPTS = 180;
```

Comment:
Please replace all types `uint` to `uint32`. In VS 2017 error: `'identifier' : undeclared identifier`

Reply:
fixed, thanks a lot!